tests/Product.test.js

```diff
@@ -7,6 +7,7 @@ const expect = chai.expect;
 chai.use(chaiHttp);
 chai.should();
 const token = process.env.TOKEN;
```

Comment:
We can have beforeEach() which we can use to get the TOKEN, 

But we will this the QA team, worries